ionoscloud/resource_application_loadbalancer_forwardingrule_test.go

```diff
@@ -165,7 +165,7 @@ func testAccCheckApplicationLoadBalancerForwardingRuleDestroyCheck(s *terraform.
 		logApiRequestTime(apiResponse)
 
 		if err != nil {
-			if apiResponse == nil || apiResponse.StatusCode != 404 {
+			if errorBesideNotFound(apiResponse) {
```

Comment:
I think you can just use httpNotFound == false? Or am I missing something?

Reply:
ah, fair :rofl: